content/scalability/docs/workloads.md

```diff
@@ -96,4 +96,6 @@ request:
 
 Mutating webhooks can modify resources in frequent succession. If you have 5 mutating webhooks and deploy 50 resources etcd will store all versions of each resource until compaction runs—every 5 minutes—to remove old versions of modified resources. In this scenario when etcd removes superseded resources there will be 200 resource version removed from etcd and depending on the size of the resources may use considerable space on the etcd host until defragmentation runs every 15 minutes.
 
+Webhook endpoint under heavy load may take longer to respond to incoming requests. This will add latencies in the response times of the cluster. This can be influenced by factors such as the number of concurrent requests being handled, the resources available to the endpoint, and any rate limiting or throttling mechanisms in place. Review and optimize the performance of the webhook endpoint code to minimize response times. This may involve identifying and addressing bottlenecks in the code, optimizing database queries, and caching frequently accessed data.
```

Comment:
We should add a paragraph that suggest CX to monitor the webhook latency.
And we should also suggest CX to use objectSelector (https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/#matching-requests-objectselector) whenever possible.

Reply:
I have added to this based on above suggestion. Please further review